client/src/components/common/boost/BoostConfigList.module.css

```diff
@@ -5,3 +5,10 @@
 .file_icon {
   transform: scale(1.5);
 }
+
+.file_name_display {
+  opacity: 0.6;
+  font-style: italic;
+  font-size: x-small;
+  text-align: center;
+}
```

Comment:
I live the idea of using `margin-left`, I just slightly increased the margin size. However, I have also kept `font-size` property because having the `fileName` just as large as the `displayName` makes it look really crowded (and I was using a laptop, on a mobile phone it might not appear user friendly imho). If you think it's too small we can probably increase the font size, just let me know.

Reply:
Nah that's fine :)